rim/static/rim/css/search.css

```diff
@@ -0,0 +1,9 @@
+.search_table {
+    padding: 3em;
```

Comment:
Pretty sure you can just use [`grid-padding-x`](https://foundation.zurb.com/sites/docs/xy-grid.html#gutters) and/or [`grid-padding-y`](https://foundation.zurb.com/sites/docs/xy-grid.html#gutters) instead of making your own classes